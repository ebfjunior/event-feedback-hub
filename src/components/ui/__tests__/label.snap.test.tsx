import React from 'react';
import { render } from '@testing-library/react';
import { Label } from '../label';

describe('ui/Label (snapshot)', () => {
	it('renders', () => {
		const { container } = render(<Label htmlFor="x">Label</Label>);
		expect(container.firstChild).toMatchSnapshot();
	});
});


