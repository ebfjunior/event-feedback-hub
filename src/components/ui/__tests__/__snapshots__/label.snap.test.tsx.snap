// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ui/Label (snapshot) > renders 1`] = `
<label
  class="flex items-center gap-2 text-sm leading-none font-medium select-none group-data-[disabled=true]:pointer-events-none group-data-[disabled=true]:opacity-50 peer-disabled:cursor-not-allowed peer-disabled:opacity-50"
  data-slot="label"
  for="x"
>
  Label
</label>
`;
